packages/hashi/src/kolibri.js

```diff
@@ -92,6 +104,9 @@ export default class Kolibri extends BaseShim {
        * @param {Object} options - The different options to filter by
        * @param {string=} options.parent - id of the parent node to filter by, or 'self'
        * @param {string[]} options.ids - an array of ids to filter by
+       * @param {string[]} options.authors - an array of authors to filter by
+       * @param {string[]} options.tags - an array of tags to filter by
+       * @param {string[]} options.activityType - an array of activityType to filer by
```

Comment:
Since it's an array and not a single activity type, the name should be in plural: `options.activityTypes`.